packages/wallets/src/features/accounts/screens/IDVDocumentUpload/components/IDVDocumentUploadDetails/IDVDocumentUploadDetails.tsx

```diff
@@ -65,6 +95,18 @@ const IDVDocumentUploadDetails = () => {
                     <SideNote />
                 </div>
             </div>
+            <div
+                className={classNames('wallets-idv-document-details__checkbox', {
+                    'wallets-idv-document-details__checkbox--disabled': !isValid,
+                })}
+            >
+                <Field disabled={!isValid} id='idv-checkbox' name='verifiedIdvDetails' type='checkbox' />
+                <label htmlFor='idv-checkbox'>
+                    <WalletText lineHeight='2xs' size='sm'>
+                        I confirm that the name and date of birth above match my chosen identity document
+                    </WalletText>
+                </label>
```

Comment:
Formik Field with boolean form value used instead of useState to be fetched as a requirement in Verification. This enables the 'Next' button without having to pass states across components

cc: @adrienne-deriv

Reply:
reference: https://github.com/jaredpalmer/formik/issues/2059